test/extensions/filters/common/rbac/mocks.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include "extensions/filters/common/rbac/engine.h"
+
+#include "gmock/gmock.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Filters {
+namespace Common {
+namespace RBAC {
+
+class MockEngine : public RoleBasedAccessControlEngine {
+public:
+  MockEngine(){};
```

Comment:
nit: for perf reasons constructor/destructor should go in cc file (see other examples, gmock docs).

Reply:
👍 

(For any code archeologists in the future: https://github.com/google/googletest/blob/master/googlemock/docs/CookBook.md#making-the-compilation-faster)